portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SearchAPI.testcase

```diff
@@ -143,6 +158,65 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-12058 | Automation ID: LRAC-12145 | Test Summary: Fetch keywords with page parameter"
+	@priority = "3"
+	test RequestKeywordsUsingPage {
+		property test.name.skip.portal.instance = "SearchAPI#RequestKeywordsUsingPage";
+
+		var parametersFirstPage = "size=${size}&displayLanguageId=en-US&page=${firstPage}&groupId=${groupId}&sort=counts,desc,lastModifiedDate,desc";
+		var parametersSecondPage = "size=${size}&displayLanguageId=en-US&page=${secondPage}&groupId=${groupId}&sort=counts,desc,lastModifiedDate,desc";
+		var groupId = JSONGroupSetter.setGroupId(groupName = "Guest");
```

Comment:
Hey @manuferreira, I think you don't need to use the `groupId` here. What do you think?